packages/fetchye/package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "fetchye",
+  "version": "1.0.0",
+  "description": "If you know how to use Fetch, you know how to use Fetchye [fetch-yae]. Simple React Hooks, Centralized Cache, Infinitely Extensible.",
+  "license": "Apache-2.0",
+  "main": "lib/index.cjs.js",
+  "module": "lib/index.esm.js",
+  "browser": "lib/index.umd.js",
+  "repository": "americanexpress/fetchye",
+  "bugs": {
+    "url": "https://github.com/americanexpress/fetchye/issues"
+  },
+  "homepage": "https://github.com/americanexpress/fetchye#readme",
+  "files": [
+    "lib"
+  ],
+  "scripts": {
+    "clean": "rimraf lib",
+    "prebuild": "yarn clean",
+    "build": "rollup -c"
+  },
+  "keywords": [
+    "async",
+    "react",
+    "redux",
+    "react-redux",
+    "hooks",
+    "ssr",
+    "fetch",
+    "data",
+    "query",
+    "promise"
+  ],
+  "author": "Michael A Tomcal <michael.a.tomcal@aexp.com>",
+  "contributors": [
+    "James Singleton <James.Singleton1@aexp.com> https://github.com/JamesSingleton)",
+    "Jimmy King  <Jimmy.King@aexp.com> (https://github.com/10xLaCroixDrinker)",
+    "Jonathan Adshead <Jonathan.Adshead@aexp.com> (https://github.com/JAdshead)",
+    "Michael Tobia <Michael.M.Tobia@aexp.com> (https://github.com/Francois-Esquire)",
+    "Michael Tomcal <Michael.A.Tomcal@aexp.com> (https://github.com/mtomcal))",
+    "Nelly Kiboi <Nelly.J.Kiboi@aexp.com> (https://github.com/nellyk)",
+    "Nickolas Oliver <nickolas.oliver@aexp.com> (https://github.com/PixnBits)",
+    "Andrew Curtis <andrew.curtis@aexp.com> (https://github.com/drewcur)",
+    "Scott McIntyre <scott.mcintyre@aexp.com> (https://github.com/smackfu)"
+  ],
+  "dependencies": {
+    "@babel/runtime": "^7.11.2",
+    "create-shared-react-context": "^1.0.3",
+    "object-hash": "^2.0.3",
+    "prop-types": "^15.7.2"
+  },
+  "peerDependencies": {
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1"
+  },
+  "optionalDependencies": {
+    "immutable": "^4.0.0-rc.12",
```

Comment:
why is immutable still optional ?

Reply:
I think the same for all optionals. That's for the different packages to bring in, no?